mail/notmuch/Portfile

```diff
@@ -34,11 +35,13 @@ depends_build       port:pkgconfig \
                     port:py37-sphinxcontrib-serializinghtml
 
 depends_lib         port:gmime3 \
+                    port:gpgme \
                     port:talloc \
                     port:xapian-core \
                     port:zlib
 
-patchfiles          ${distname}-doc-makefile.patch \
+patchfiles          ${distname}-configure.patch \
```

Comment:
I accept this is largely a matter if taste, so take this comment as is, but for small patches I tend to prefer using a  reinplace regex replacement to a patch file. I find it neater, as it avoids an additional file, and also is usually more robust to future changes in the patched file.

Reply:
My goal is to have a patch/fix accepted upstream, hence the (versioned via its name) patch.

Reply:
Ok, a very reasonable reason...